pkg/btf/btf.go

```diff
@@ -89,14 +88,13 @@ func InitCachedBTF(ctx context.Context, lib, btf string) error {
 	if err != nil {
 		return fmt.Errorf("tetragon, aborting kernel autodiscovery failed: %w", err)
 	}
-	btfObj, err = NewBTF()
 	return err
 }
 
 func GetCachedBTFFile() string {
 	return btfFile
 }
 
-func GetCachedBTF() *btf.Spec {
-	return btfObj
+func GetCachedBTF() (*btf.Spec, error) {
```

Comment:
Could we just remove this function altogether and have callers use NewBTF, since we do not cache the object anymore?

Reply:
sure I was just minimizing changes.